### PR TITLE
JavaScript

### PR DESCRIPTION
**Pull request by emtec-jitendra-patil**
Completed All  Assignment's of JavaScript.

### CODE REVIEW

File: JavaScript/JavascriptAssignment's.js [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+// -----------------------------Question - 5--------------------------------
+// Create a function that takes the age in years and returns the age in days.
+//  calcAge(65) ➞ 23725
+//  calcAge(0) ➞ 0
+//  calcAge(20) ➞ 7300
+// ------------------------------Solution----------------------------------
+
+// function calcAge(years){
+//     let getAge = years * 365;
```
Comment (rishikesh-dhanawade-emtec) [inline JavaScript/JavascriptAssignment's.js:74]: if there is leap year between, can we check this condition

**Review by rishikesh-dhanawade-emtec — COMMENTED** [review]
please check comments

File: JavaScript/JavascriptAssignment's.js (hunk 1 above)
Comment (emtec-jitendra-patil) [reply JavaScript/JavascriptAssignment's.js:74]: function calcAge(years){
    let leapDays = (years / 4)
    let getAge = (years * 365) + leapDays;
    return getAge;
}

Comment (rishikesh-dhanawade-emtec) [inline JavaScript/JavascriptAssignment's.js:74]: what if there is leap year in between? can we write logic for that

**Review by rishikesh-dhanawade-emtec — CHANGES_REQUESTED** [review]
check comment

**Review by rishikesh-dhanawade-emtec — APPROVED** [review]
approved

**Pull request merged**